soccer_ipm/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>soccer_ipm</name>
+  <version>0.0.0</version>
+  <description>Inverse perspctive mapping for the RoboCup soccer environment</description>
+  <maintainer email="florian@flova.de">Florian Vahl</maintainer>
+  <license>Apache License 2.0</license>
+
+  <exec_depend>soccer_vision_2d_msgs</exec_depend>
+  <exec_depend>soccer_vision_3d_msgs</exec_depend>
+  
+  <depend>cv_bridge</depend>
+  <depend>geometry_msgs</depend>
+  <depend>ipm_library</depend>
+  <depend>python3-opencv</depend>
+  <depend>sensor_msgs_py</depend>
+  <depend>sensor_msgs</depend>
+  <depend>std_msgs</depend>
+  <depend>tf2_geometry_msgs</depend>
+  <depend>tf2_sensor_msgs</depend>
+  <depend>tf2</depend>
+  <depend>vision_msgs</depend>
```

Comment:
Just wondering why some dependencies are ``<depend>`` and the others are ``<exec_depend>``. I believe best practice is to make all dependencies an ``<exec_depend>`` in python packages.

Reply:
Some also required for testing etc. is that included in exec depend?